lib/gooddata/models/metadata/report.rb

```diff
@@ -31,6 +31,11 @@ def create(options = { :client => GoodData.connection, :project => GoodData.proj
         fail 'Report needs a title specified' unless title
         summary = options[:summary] || ''
 
+        if options[:folder]
```

Comment:
Refers to 'options' more than self (maybe move it to another class?) (FeatureEnvy)